packages/common/src/api/account.ts

```diff
@@ -162,7 +154,6 @@ const accountApi = createApi({
         return payload
       },
       options: {
-        idArgKey: 'managerUserId',
```

Comment:
I continue to not be entirely certain how this works. I seem to remember either the returned response being the wrong shape or the nested items not being cached correctly when I did not set this. Do you mind doing a verification pass on manager mode to ensure this didn't break anything?

Reply:
yeah happy to! You're probably thinking of `schemaKey` though. `idArgKey` is only used to hit the legacy cache and shouldn't impact behavior when it misses

https://github.com/AudiusProject/audius-protocol/blob/main/packages/common/src/audius-query/README.md#enable-single-entity-cache-hits